testsuite/classlibrary/TestUGen_Duty.sc

```diff
@@ -21,8 +29,11 @@ TestUGen_Duty : UnitTest {
 				values.stutter(numFrames),
 				"Duty should output exact values at audio rate. Tested with % frames per value".format(numFrames)
 			);
+			cond.test_(true).signal;
 		};
 		{ func.value(dt, values) }.loadToFloatArray(dur, server, action);
+
+		cond.wait;
```

Comment:
... and this as `hang`

I find the combination of `hang` and `unhang` easier to read. But both is fine with me!

Reply:
OK, I find this combination easier to read, since wait/signal is the standard interface terminology for condition variables. I have some thoughts about the Condition interface in general, will send on the ML soon.